### PR TITLE
[WIP] Updating to v6 core 1685

### LINKED ISSUE #25: Update to Core v6
Update the host to core v6

### PR DESCRIPTION
**Pull request by andreasohlund**
Closes #25 

Just got it compiling

### CODE REVIEW

**Comment by andreasohlund** [comment]
@Particular/azure-maintainers I got this one compiling, need help with testing it

File: src/NServiceBus.Hosting.Azure.HostProcess/app.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
 
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5" /></startup>
+<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1" /></startup>
```
Comment (SimonCropp) [inline src/NServiceBus.Hosting.Azure.HostProcess/app.config]: remove this

File: src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>NServiceBus.Hosting.Azure.HostProcess</RootNamespace>
     <AssemblyName>NServiceBus.Hosting.Azure.HostProcess</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```
Comment (SimonCropp) [inline src/NServiceBus.Hosting.Azure.HostProcess/NServiceBus.Hosting.Azure.HostProcess.csproj]: we target 4.5.2 not 4.6.1

**Comment by SimonCropp** [comment]
nuspecs still need updating

**Comment by SimonCropp** [comment]
remove the binding redirect in the nuget. it adds no value

**Comment by yvesgoeleven** [comment]
@andreasohlund thank you! I wanted to start this today, after I had all the PR reviews done. So I'll help out testing it..

**Comment by yvesgoeleven** [comment]
Tested it with hostproc & roleentrypoint hosted endpoint, merging in

**Pull request merged**